### PR TITLE
initial commit; You can set the max input length of the search-input.

### PR DESCRIPTION
**Pull request by GergelyLakatos**
I fixed the issue what i opened.  (https://github.com/charliekassel/vuejs-autocomplete/issues/44)
I hope you accept it. Please notice me if something wrong with it.

### CODE REVIEW

File: src/components/Autocomplete.vue [hunk 1]
```diff
@@ -149,6 +150,14 @@ export default {
      */
     clearButtonIcon: {
       type: String
+    },
+
+    /**
+     * Optional max input length
+     */
+    maxlength: {
+      type: Integer,
```
Comment (charliekassel) [inline src/components/Autocomplete.vue]: this should be `Number`

Comment (GergelyLakatos) [reply src/components/Autocomplete.vue]: you right! I updated it. :)

**Comment by GergelyLakatos** [comment]
I removed the default value, and corrected the README file.

**Comment by GergelyLakatos** [comment]
Thank you! :)

**Pull request merged**